packages/app-store/routing-forms/trpc/response.handler.ts

```diff
@@ -99,14 +99,13 @@ export const responseHandler = async ({ ctx, input }: ResponseHandlerOptions) =>
 
     const settings = RoutingFormSettings.parse(form.settings);
     let userWithEmails: string[] = [];
-    if (form.teamId && settings?.sendUpdatesTo?.length) {
+    if (form.teamId && (settings?.sendToAll || settings?.sendUpdatesTo?.length)) {
+      const whereClause: Prisma.MembershipWhereInput = { teamId: form.teamId };
+      if (!settings?.sendToAll) {
+        whereClause.userId = { in: settings.sendUpdatesTo };
+      }
```

Comment:
Instead of doing this, shouldn't we find and update sendUpdatesTo to include future members? Why are we currently not adding future members to it?

Reply:
Just wanted to avoid having an prisma update request on `App_RoutingForms_Form` table every time there is a response on the routing Form. (in cases of sendToAll)

If we take UI approach to update `sendUpdatesTo` when switch is selected in RoutingForm, we may miss on the scenario where - new team member is added after the switch in RoutingForm is selected and the sendUpdatesTo wont have future members until the switch is re-toggled.

Reply:
IMO , even if we keep update `sendUpdatesTo` in sync, there is no advantage from this. Because any how, here in `response.handler.ts` we need to check for latest new members added if any.
I understand. that's the reason we have an additional variable `sendToAll`.

In that case, we will be making an unnecessary update operation for each response.